docs/user/alerting/action-variables.asciidoc

```diff
@@ -5,405 +5,230 @@
+[[rule-name-variable]]`rule.name`::
+(string) The rule name.
+`rule.params`::
+(object) The rule parameters, which vary by <<rule-types,rule type>>.
+[[rule-spaceid-variable]]`rule.spaceId`::
+(string) The space identifier for the rule. For example, `default`.
+[[rule-tags-variable]]`rule.tags`::
+(array) The list of tags applied to the rule.
+`rule.type`::
+(string) The type of rule. For example, `.es-query`.
+`rule.url`::
+(string) The URL for the rule that generated the alert. This will be an empty string if the `server.publicBaseUrl` setting is not configured.
+//TBD Is it an empty string or omitted (latter per my test)?
+`spaceId`::
+(string) deprecated:[8.10.0] Use <<rule-spaceid-variable,`rule.spaceId`>> instead.
+`tags`::
+(array) deprecated:[8.10.0] Use <<rule-tags-variable,`rule.tags`>> instead.
 
 [float]
 [role="child_attributes"]
 [[alert-summary-action-variables]]
-==== Action frequency: Summary of alerts
+=== Action frequency: Summary of alerts variables
 
 If the rule's action frequency is a summary of alerts, it passes the following variables:
 
-`alerts.all.count`:: The count of all alerts.
+`alerts.all.count`::
+(integer) The count of all alerts.
 
 `alerts.all.data`::
-An array of objects for all alerts. The following object properties are examples; it is not a comprehensive list.
+(array) An array of objects for all alerts.
+The following object properties are examples; it is not a comprehensive list.
 +
 .Properties of the alerts.all.data objects
 [%collapsible%open]
 =====
 //# tag::alerts-data[]
-`kibana.alert.end`:: Datetime stamp of alert end. preview:[]
-`kibana.alert.flapping`:: A flag on the alert that indicates whether the alert status is changing repeatedly. preview:[]
-`kibana.alert.instance.id`:: ID of the source that generates the alert. preview:[]
-`kibana.alert.reason`:: The reason of the alert (generated with the rule conditions). preview:[]
-`kibana.alert.start`:: Datetime stamp of alert start. preview:[]
-`kibana.alert.status`:: Alert status (for example, active or OK). preview:[]
+`_id`::
+(string) For example, `27ace796-1dfb-4db9-9ce1-7a10472fd461`.
+// TBD: Is this the same as the alert.uuid?
+`_index`::
+(string) For example, `.internal.alerts-stack.alerts-default-000001`.
+// TBD: The index that contains the alert data.
+`@timestamp`::
+(string) For example, `2024-04-30T02:30:11.254Z`.
+// TBD: The timestamp that the alert was added to the index?
+`event.action`::
+(string) For example: `active`.
+// TBD: Is this the same as the alert.status?
+`event.kind`::
+(string) For example: `signal`.
+// TBD: The type of event (are there types other than signal?)
+`kibana.alert.action_group`::
+(string) The ID of the action group of the alert that scheduled the action. For example, `query matched`.
+// TBD: Is this a repeat of alert.actionGroup?
+`kibana.alert.consecutive_matches`::
+(integer)
+// TBD: The number of consecutive runs that generated the alert?
```

Comment:
Yes, this is part of our conditional "only run actions if alert is active for X consecutive runs.